hardware-testing/hardware_testing/scripts/shenzhen_scale.py

```diff
@@ -0,0 +1,36 @@
+"""Shenzhen Scale."""
+import datetime
+from hardware_testing import data
+from hardware_testing.drivers import find_port, list_ports_and_select
+from hardware_testing.drivers.radwag import RadwagScale
+
+if __name__ == "__main__":
+    # find port using known VID:PID, then connect
+    vid, pid = RadwagScale.vid_pid()
+    try:
+        scale = RadwagScale.create(port=find_port(vid=vid, pid=pid))
+    except RuntimeError:
+        device = list_ports_and_select()
+        scale = RadwagScale.create(device)
+    scale.connect()
+    grams, is_stable = scale.read_mass()
+    print(f"Scale reading: grams={grams}, is_stable={is_stable}")
+    test_name = "TC_Evaporation"
+    run_id = data.create_run_id()
+    tag = ""
+    file_name = data.create_file_name(test_name, run_id, tag)
+    header = ["Date", "Scale Reading", "Stable"]
+    header_str = ",".join(header) + "\n"
+    data.append_data_to_file(
+        test_name=test_name, run_id=run_id, file_name=file_name, data=header_str
+    )
+    results_list = []
+    while is_stable is False:
+        grams, is_stable = scale.read_mass()
+        print(f"Scale reading: grams={grams}, is_stable={is_stable}")
+        time_now = datetime.datetime.now()
+        row = [time_now, grams, is_stable]
+        results_list.append(row)
+        if is_stable is True:
+            print(f"Scale stable reading {grams}.")
+            break
```

Comment:
Might be nice to let the script just continue running, instead of needing to restart for every reading. Add the following 2x lines would do that I think:
```suggestion
            ui.get_user_ready("to measure weight again")
            is_stable = False
```

Reply:
I added a few more lines of code to ensure that the file save data would be different from the previous sample. 
`y_or_no = input("Do you want to weight another sample? (Y/N): ")
            if y_or_no == "Y":
                is_stable = False
                save_file_variables = _create_scale_reading_file()
            elif y_or_no == "N":
                break_all = True
        if break_all:
            break`